scripts/jenkins/ardana/ansible/deploy-crowbar.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- import_playbook: cloud-ses-integration.yml
```

Comment:
why do we need this twice (in deploy-cloud  and deploy-crowbar)?

Reply:
deploy-cloud is for Ardana, whereas deploy-crowbar is for Crowbar.